books/models.py

```diff
@@ -1,13 +1,17 @@
 from django.db import models
 from django.urls import reverse
+from django.contrib.auth import get_user_model
+
+CustomUser = get_user_model()
+superuser_id = CustomUser.objects.filter(is_superuser=True)[0].id
```

Comment:
I missed this before, but saw it now that I came to check.
A couple of problems with this:
1. There is no guarantee that a superuser will exist, so this will break **before the app loads** for someone who has just installed the app.
2. All ORM queries should be "lazy". That means you should not be performing them before Django finishes loading all models. In practice it means you should not use the ORM in models unless it's within functions/methods.
So these are:

good:
```python
class Book(models.Model):
    def get_default_owner():
        return CustomUser.objects.filter(...)
```

bad:
```python
class Book(models.Model):
    owner = models.ForeignKey(...
        default=CustomUser.objects.filter(...)
    )
```
bad:
```python
default_owner = CustomUser.objects.filter(...)

class Book(models.Model):
    owner = models.ForeignKey(...
        default=default_owner...)
    )
```

If you want to ensure that an owner is always set and you don't want it to be nullable, then you can use `on_delete=SET(...)`. There is a good example here:
https://docs.djangoproject.com/en/2.1/ref/models/fields/#django.db.models.SET